src/oop/user_defined_classes/medium/q4/QuestionText.md

```diff
@@ -0,0 +1,21 @@
+# oop-user_defined_classes-medium-q4
+
+## Java
+
+We always get thirsty in the summer, and it is important to stay hydrated. Make a Java
+class called `Water` that has two attributes called **volume** and **quality** with a constructor
+that initializes them.
+
+## Water
+
+The `Water` class you are given has the following items:
+
+- Attributes:
+    - A `double` called **volume** that is not accessible from another class.
+    - A `String` called **quality** that is not accessible from another class.
+
+You need to add the following items:
+
+- Constructor:
+    - Create a constructor that initializes these attributes in that order. Also, if the input
```

Comment:
I would specify the order again here and you also need to provide what the accessibility modifier should be